src/tests/Info.test.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Info from '../components/Info';
+
```

Comment:
- [ ] You should also write integration tests for your components with React Testing Library. (For both components)
(For example, if you click on the country, it should take you to the information page. )

From Project Specifications :arrow_down: 
>Testing requirements
>
>   Create unit tests for pure functions in your Redux code.
>   Create integration tests for your application using the React Testing Library.